src/SafetyMonitor.cpp

```diff
@@ -42,19 +44,66 @@ int main(int argc, char **argv)
    // Create a handle for this particular node, which is
    // responsible for handling all of the ROS communications
    ros::NodeHandle nh;
-   ros::NodeHandle param_nh ("iarc7_safety_node");
+   ros::NodeHandle private_nh ("~");
 
    // Create a publisher to advertise this node's presence.
    // This node should only publish in case of emergency, so queue length is 100
    // TODO : Change std_msgs::String to a custom type
    ros::Publisher safety_publisher = nh.advertise<std_msgs::String>("safety", 100);
-   
+
    // Specify a time for the message loop to wait between each cycle (ms)
    ros::Rate loop_rate(kLoopFrequencyHz);
 
    // Read in parameter containing the bond table
    std::vector<std::string> bond_ids;
```

Comment:
I was reusing this vector, but I can make two separate vectors for those things.